frontend/src/test/java/com/google/tripmeout/frontend/servlet/ServletUtilTest.java

```diff
@@ -104,15 +104,13 @@ public void extractFromRequestBody_supplyPlaceVisitObject_returnsOptionalPlaceVi
       throws IOException, EmptyRequestBodyException {
     when(request.getReader())
         .thenReturn(new BufferedReader(new StringReader(
-            "{tripId: abc123, placesApiPlaceId: 123, placeName: trip1, userMark: YES, latitude: 33.2, longitude: -22.77}")));
+            "{tripId: abc123, placesApiPlaceId: 123, placeName: trip1, userMark: YES}")));
```

Comment:
This will be easier to refactor in the future if the test is updated to use a serialized payload created from an in-memory model object:

```java
PlaceVisitModel testPlace1 = ...;
when(request.getReader())
    .thenReturn(new BufferedReader(new StringReader(
        gson.toJson(testPlace1))));
```

You can then assert that the model object you get from ServletUtil is equal to the one you originally serialized.